### PR TITLE
feat(Icon): add Track icon

### PR DESCRIPTION
**Pull request by sooinc**
### Overview

- add Track icon

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/go1LZq5mhdpWQgBzQJfxpI/Career-Journey?node-id=265%3A3926
- [x] Related to JIRA ticket: Egg-2336
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-icons/src/svg/regular/track-icon.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g transform="matrix(1,0,0,1,0,0)"><defs><style>.a{fill:none;stroke:#000000;stroke-linecap:round;stroke-linejoin:round;stroke-width:1.5px;}</style></defs><title>business-ladder</title><line class="a" x1="0.75" y1="23.25" x2="10.5" y2="0.75"></line><line class="a" x1="13.5" y1="23.25" x2="23.25" y2="0.75"></line><line class="a" x1="8.55" y1="5.25" x2="21.3" y2="5.25"></line><line class="a" x1="6.6" y1="9.75" x2="19.35" y2="9.75"></line><line class="a" x1="4.65" y1="14.25" x2="17.4" y2="14.25"></line><line class="a" x1="2.7" y1="18.75" x2="15.45" y2="18.75"></line></g></svg>
```
Comment (dreamwasp) [inline packages/gamut-icons/src/svg/regular/track-icon.svg:1]: if you set stroke:"currentColor" it will allow the color of the icon to be changed

**Review by dreamwasp — APPROVED** [review]
✨

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.1.2-alpha.590269.0<br/>@codecademy/gamut-kit@0.6.77-alpha.590269.0<br/>@codecademy/gamut-labs@30.2.3-alpha.590269.0<br/>@codecademy/gamut@45.1.2-alpha.590269.0<br/>@codecademy/monaco@0.6.23-alpha.590269.0<br/>@codecademy/styleguide@55.1.3-alpha.590269.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://622a461990394817325b8744--gamut-preview.netlify.app>https://622a461990394817325b8744--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/622a461990394817325b8744>Deploy Logs</a></p>

**Pull request merged**